src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -70,20 +76,55 @@ const AssetContainer: React.FC<Props> = ({
     { key: "name", label: intl.formatMessage({ defaultMessage: "Alphabetical" }) },
   ];
 
+  const handleRemove = useCallback(() => {
+    if (selectedAssets?.length) {
+      onRemove?.(selectedAssets);
+    }
+  }, [onRemove, selectedAssets]);
+
+  const listRef = useRef<HTMLDivElement>(null);
+  const [containerWidth, setContainerWidth] = useState(0);
+
+  useEffect(() => {
+    if (listRef.current?.offsetWidth) {
+      setContainerWidth(listRef.current?.offsetWidth);
+    }
+  }, [containerWidth]);
+
+  const handleResize = () => {
+    if (listRef.current?.offsetWidth) {
+      setContainerWidth(listRef.current?.offsetWidth);
+    }
+  };
+
+  window.addEventListener("resize", handleResize);
```

Comment:
use `useEffect` to attach a event in React components. Otherwise, each time the component is rendered, a new callback will be registered.

```ts
useEffect(() => {
  const handleResize = () => {
    if (listRef.current?.offsetWidth) {      
      setContainerWidth(listRef.current?.offsetWidth);
    }
  };
  window.addEventListener("resize", handleResize);
  return () => window.removeEventListener("resize", handleResize);
}, []);
```

Reply:
yes, I hope a new callback be registered everytime the component is rendered. because it need a initialization Calculation for the size... so... any suggestion? @rot1024

Reply:
The code I've shown you should work just fine.

Reply:
thanks, I will use another way.